### PR TITLE
Features/port workshops

### PR DESCRIPTION
**Pull request by hikes101**
I ported the workshops page and set up a basic layout. I would like to merge these changes now so I can finish porting the learn page.

### CODE REVIEW

File: views/workshops.hbs [hunk 1]
```diff
@@ -1,3 +1,41 @@
-<div class="text-center">
-    <h1>Workshops</h1>
-</div>
\ No newline at end of file
+<div class="content">
```
Comment (pheeria) [inline views/workshops.hbs]: You shouldn't need this div. It is in main.hbs

File: data/workshops.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "link4": "http://www.schulfrei-bewegung.de/"
+    },
+    {
+      "name": "Mathe x Kunst Projekt",
+      "description": "Das Ziel des Projekts liegt darin, Formate für Workshops und Schulen zu entwickeln, die sich am Schnittpunkt von Mathe/NaWi und Kunst bewegen. Wir wollen einen spielerischen, sinnlichen und anschaulichen Zugang zu den scheinbar trockenen und ungeliebten „Fächern“ Mathematik und Naturwissenschaften wagen.",
+      "link1": "https://www.mathe-x-kunst.de/",
+      "additional": "Wir suchen momentan weitere WorkshopleiterInnen...hast du Lust mitzumachen? Dann melde dich bei uns: info@mathe-x-kunst.de!"
+    }
+  ]
+}
```
Comment (pheeria) [inline data/workshops.json]: Objects should have the same pattern. If you need more links, use an array :)

File: views/workshops.hbs [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+        <a href="{{link1}}" target="_blank">
+          <h5 class="card-title">{{name}}</h5>
+        </a>
+        <p class="card-text">{{description}}</p>
+        <a class="workshops-links" href={{link2}} target="_blank">{{link2}}</a>
+        <a class="workshops-links" href={{link3}} target="_blank">{{link3}}</a>
+        <a class="workshops-links" href={{link4}} target="_blank">{{link4}}</a>
+        <p class="card-text">{{additional}}</p>
+      </div>
+      {{/each}}
```
Comment (pheeria) [inline views/workshops.hbs]: Please, extract this to a partial and add a snapshot test

File: src/routes/workshops.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
 const express = require("express");
+const fs = require("fs");
 
 const router = express.Router();
 
 router.get("/workshops", (req, res) => {
-  res.render("workshops");
+  fs.readFile("data/workshops.json", (err, data) => {
```
Comment (pheeria) [inline src/routes/workshops.js]: Can you please do it the same way like in Courses and Home routers?

File: views/partials/workshops/seminars.hbs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<div class="card">
+  <a href="{{mainLink}}" target="_blank">
+    <h5 class="card-title">{{name}}</h5>
+  </a>
+  <p class="card-text">{{description}}</p>
+  <a class="workshops-links" href={{link1}} target="_blank">{{link1}}</a>
+  <a class="workshops-links" href={{link2}} target="_blank">{{link2}}</a>
+  <a class="workshops-links" href={{link3}} target="_blank">{{link3}}</a>
+  <p class="card-text">{{additional}}</p>
+</div>
```
Comment (pheeria) [inline views/partials/workshops/seminars.hbs]: Please, name this file in singular, without **s** - to match the other partials :)

File: src/routes/workshops.js (hunk 4 above)
Comment (hikes101) [reply src/routes/workshops.js]: @pheeria Done

**Pull request merged**